contracts/core/EntryPoint.sol

```diff
@@ -518,6 +526,9 @@ contract EntryPoint is IEntryPoint, StakeManager, NonceManager, ReentrancyGuard,
             } catch {
                 revert FailedOpWithRevert(opIndex, "AA33 reverted", Exec.getReturnData(REVERT_REASON_MAX_LEN));
             }
+            if (preGas - gasleft() > pmVerificationGasLimit) {
+                revert FailedOp(opIndex, "AA36 over verificationGasLimit");
```

Comment:
The revert message is identical to the account's case (except for AA36 vs. AA26).  Shouldn't it say "over paymasterVerificationGasLimit"?